### ISSUE TITLE
[question] - Paste doesn't work in the trial version

### ISSUE THREAD

**Issue by furaiev** [AvdLee/RocketSimApp#107]
Hello, 
Should the 'paste' option work in the trial version?
It is always disabled but I have an image in the buffer for sure.

**Comment by AvdLee** [comment]
> Hello,
> Should the 'paste' option work in the trial version?
> It is always disabled but I have an image in the buffer for sure.

Yes, it should work. Which type of image do you have in your clipboard and how did you copy it?

**Comment by furaiev** [comment]
I copied the image as .png in Figma (right like in your examples), as a result, I paste it everywhere but not in the RocketSimApp.
Also, I've tried to copy just a .png file from the mac finder - the same result.

**Comment by AvdLee** [comment]
Can you share me a diagnostics report? 

Preferences -> About -> Send Diagnostics

**Comment by AvdLee** [comment]
I've found the issue, Figma is no longer returning PNG but instead TIFF:

```
Pasteboard item with types [__C.NSPasteboardType(_rawValue: public.tiff)]
```

I'm going to update my code to handle any image type 🙂

**Comment by AvdLee** [comment]
This is fixed in 5.0.1
